testsuite/integration/clustering/pom.xml

```diff
@@ -832,6 +832,42 @@
                                     </systemPropertyVariables>
                                 </configuration>
                             </execution>
+
+                            <execution>
+                                <id>ts.surefire.clustering.web-passivation</id>
+                                <!-- Disabled by default; other profiles turn it on -->
+                                <phase>none</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <!-- Tests to execute. -->
+                                    <includes>
+                                        <include>org/jboss/as/test/clustering/cluster/web/**/CoarseSessionPassivationTestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/web/**/FineSessionPassivationTestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/web/**/CoarseTransactionalSessionActivationTestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/web/**/NonDistributableTestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/web/**/FineTransactionalSessionActivationTestCase.java</include>
```

Comment:
Hello @pferraro @rhusar ,

Could you point me out to any missing test that should be included for this Galleon layer? The above list are the tests I found suitable to work with this kind of configuration but I want to be sure I'm not missing anything, I'm not very familiar with the clustering test suite. Thanks!

Reply:
Everything within org/jboss/as/test/clustering/cluster/web/** should be testable with this profile.

Reply:
@pferraro Notice that the web-passivation Galleon layer don't provision `jgroups` and configures a local infinispan web container cache. My understanding is the web session is not shared across all the cluster members, most of the tests under org/jboss/as/test/clustering/cluster/web/** assert values for all the cluster nodes.

Reply:
@pferraro @rhusar 
I've adapted the code to use the single arquillian launch. Since the web-passivation Galleon layer uses a local infinispan web container cache, jgroups is not provisioned, that means, we cannot use any of the available tests that requires a cluster.

That made me to identify only one test available for a single node configuration [SimpleWebTestCase](https://github.com/wildfly/wildfly/blob/master/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/single/web/SimpleWebTestCase.java). Could you let me know if that test would be enough to test the capabilities giving by the web-passivation Galleon layer or, by the contrast, which other tests available on the clustering test suite are suitable to test this layer?

Reply:
This test should be used to validate this configuration: https://github.com/wildfly/wildfly/blob/master/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/NonHaWebSessionPersistenceTestCase.java

Reply:
ok, thank you Paul!